packages/flutter/lib/src/widgets/text.dart

```diff
@@ -477,6 +492,12 @@ class Text extends StatelessWidget {
   /// [MediaQuery], or 1.0 if there is no [MediaQuery] in scope.
   final double? textScaleFactor;
 
+  /// An optional minimum number of lines for the text to span.
+  ///
+  /// If the text falls short of the given number of lines, lines will be added
+  /// below until there are the given number of lines.
+  final int? minLines;
```

Comment:
What's the difference between `minLines == null` and `minLines == 0`?

Reply:
We can make `minLines` default to 0, there is no behavioural difference.